src/controller/CHIPDeviceController.h

```diff
@@ -713,16 +736,24 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
      */
     CHIP_ERROR ProcessOpCSR(const ByteSpan & NOCSRElements, const ByteSpan & AttestationSignature);
 
+    CHIP_ERROR ProcessCertificateChain(const ByteSpan & certificate);
+
+    CHIP_ERROR ValidateAttestationInfo(chip::ByteSpan attestationElements, chip::ByteSpan signature);
```

Comment:
Please add some documentation to these methods.

Also, consider making the arguments `const ByteSpan &`

Reply:
will update